benchmarks/bench_evolve.py

```diff
@@ -13,6 +13,41 @@
 import pineko.theory_card
 
 
+@pytest.mark.parametrize("theoryid,is_mixed", [(400, False), (208, True)])
```

Comment:
Good point, I did not realize. I usually compute 7 but for production would be good to be aware of this @giacomomagni

Reply:
I'm not sure how you generate eko runcards from the DIS grids, or how the convolution of the two is done under the hood. The one thing to consider is that (as Giacomo pointed out) points that vanish in the DIS grid don't are not included in the eko. So to have a general eko that can be used for all fktables at a given nf you'd probably want to include the `mugrid` based on the dataset rather than the DIS grid - that, or you'd somehow need to merge the DIS grids to get the union of all non-zero points. 

Either way - if this sounds like something that could be worth it maybe open an issue and keep it for a separate PR.

Reply:
Yes I agree, for the moment we can generate one eko for each subtheory and that's it. Later on we can surely improve both this and the inputs issue @AleCandido pointed out

Reply:
Yes you are right, the ekos do not need to be splatted... 
But I prefer having something automatised as possible, so a solution would be that if you have 7 theories, *02 and *03 (*05, *06) share the same ekos.

Reply:
#130